views/layouts/main.handlebars

```diff
@@ -7,39 +7,56 @@
+    <link rel="icon" href="/images/botomir.jpg"> <!-- Need .ico file type -->
 
     <!-- Bootstrap core CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap.min.css"
           integrity="sha384-VCmXjywReHh4PwowAiWNagnWcLhlEJLA5buUprzK8rxFgeH0kww/aWY76TfkUoSX" crossorigin="anonymous">
+
+    <!-- Font Awesome CDN -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- Custom CSS-->
     <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" type="text/css" href="css/about.css">
+    <link rel="stylesheet" type="text/css" href="css/statistics.css">
 
     <!-- Font -->
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
 </head>
 <body>
-<nav class="navbar navbar-expand-md navbar-custom">
-    <a class="navbar-brand" href="/">
-        <img src="/images/logo.jpg" alt="Botomir Logo" width="30" height="24" class="d-inline-block align-top">
-        Botomir
-    </a>
+<div id="page-container">
+    <div id="content-wrap">
+        <nav class="navbar navbar-expand-md navbar-custom">
+            <a class="navbar-brand" href="/">
+                <img src="/images/botomir.jpg" alt="Botomir Logo" width="30" height="24"
+                     class="d-inline-block align-top">
+                Botomir
+            </a>
 
-    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
-        <div class="navbar-nav">
-            <a class="nav-item nav-link active" href="/documentation">Documentation<span class="sr-only">(current)</span></a>
-        </div>
-    </div>
+            <div class="navbar-collapse" id="navbarNavAltMarkup">
+<!--                <div class="navbar-nav">-->
+<!--                    <a class="nav-item nav-link active" href="/documentation">Documentation</a>-->
+<!--                </div>-->
+<!--                <div class="navbar-nav">-->
+<!--                    <a class="nav-item nav-link active" href="/statistics">Statistics</a>-->
+<!--                </div>-->
+            </div>
 
-    <div class="collapse navbar-collapse">
-        <div class="navbar-nav ml-auto">
-            <a class="nav-item nav-link" href="https://github.com/SoorajModi/Botomir" target="_blank">GitHub</a>
-        </div>
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a class="nav-item nav-link" href="/about">About</a>
+                </li>
+            </ul>
+        </nav>
+        {{{body}}}
     </div>
-</nav>
-{{{body}}}
+    <footer>
+        <a href="https://github.com/SoorajModi/Botomir" target="_blank" class="footer-icon">
+            <i class="fa fa-github fa-lg"></i>
+        </a>
+    </footer>
+</div>
 </body>
-<footer>
-    <p>GPL 2.0 License</p>
```

Comment:
Should the license / copy-write statement be kept in the footer? 
(Not a change request, but a question)

Reply:
Its not something we really need.

We should have a user agreement and privacy policy however

Reply:
This has been marked in #69